checks/libs/wmi/sampler.py

```diff
@@ -91,8 +92,10 @@ class WMISampler(object):
 
     def __init__(self, logger, class_name, property_names, filters="", host="localhost",
                  namespace="root\\cimv2", provider=None,
-                 username="", password="", and_props=[], timeout_duration=10):
+                 username="", password="", and_props=[],
+                 mute=True, timeout_duration=10):
```

Comment:
The Datadog integration checks relying on `wmi` have `mute` set `False`.  This allows the check to raise, and the exception message to be used by the `dd-agent` info page.